project/MiMa.scala

```diff
@@ -768,6 +768,12 @@ object MiMa extends AutoPlugin {
 
         // #20362 - parser private
         ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.http.impl.model.parser.CommonRules.expires-date")
+      ),
+      "2.4.5" -> Seq(
+        // #20462 - now uses a Set instead of a Seq within the private API of the cluster client
+        ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.cluster.client.ClusterClient.contacts_="),
+        ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.cluster.client.ClusterClient.contacts"),
+        ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.cluster.client.ClusterClient.initialContactsSel")
```

Comment:
this should be in the `2.4.4` section, 2.4.5 is not released yet
that is probably the why the build is still failing